src/Illuminate/Foundation/Exceptions/views/4xx.blade.php

```diff
@@ -0,0 +1,11 @@
+@extends('errors::minimal')
+
+@php
+    $statusCode = $exception->getStatusCode();
+    $statusText = __(\Symfony\Component\HttpFoundation\Response::$statusTexts[$statusCode]);
+    $statusMessage = !empty($exception->getMessage()) ? ': ' . __($exception->getMessage()) : '';
```

Comment:
We should never show raw exception messages to users.

Reply:
But that's exactly what [currently happens](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Foundation/Exceptions/views/403.blade.php#L5) in `403.blade.php` of the Framework.

So, shouldn't we do something about it?

If we agree that we shouldn't show it, I see two options:

1. Show it only if `APP_DEBUG=true`
2. Just remove it from the templates.

I prefer the first option in this case. What are your thoughts?

Thank you for your comments.